mrmustard/physics/converters.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains the functions to convert between different representations.
+"""
+from typing import Iterable, Union, Optional
+from mrmustard.physics.representations import Representation, Bargmann, Fock
+from mrmustard import math, settings
+
+
+def to_fock(rep: Representation, cutoffs: Optional[Union[int, Iterable[int]]] = None) -> Fock:
```

Comment:
I think we should use `shape` rather than `cutoffs`. Historically in MrMustard `len(cutoffs) == len(modes)`, whether you are a ket or a dm. But `len(shape)` is more general.

Reply:
I will put here shape rather than cutoffs. Would it be a little bit ambiguous since the shape would include the batch but cutoffs is just the cutoffs of the Fock and not include batch information? @ziofil